{
  'test_type': 'output_check',
  'errors': """
gh_597.pwn(5) : error 021: symbol already defined: "e0"
  """
}
